db/seeds.rb

```diff
@@ -3,11 +3,21 @@
 #
 # Examples:
 #
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-Skill.create([{ name: 'skill1', featured: true },
-              { name: 'skill2', featured: true },
-              { name: 'skill3', featured: false },
-              { name: 'skill4', featured: true },
-              { name: 'skill5', featured: false },
+#   cities = City.create([{ name: "Chicago" }, { name: "Copenhagen" }])
+#   Mayor.create(name: "Emanuel", city: cities.first)
+Skill.create([{ name: "skill1", featured: true },
+              { name: "skill2", featured: true },
+              { name: "skill3", featured: false },
+              { name: "skill4", featured: true },
+              { name: "skill5", featured: false },
+             ])
+
+Group.create([{ name: "1409" },
+              { name: "1410" },
+              { name: "1412" },
+             ])
+
+Group.create([{ name: '1409' },
+              { name: '1410' },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.